src/tools/draw.js

```diff
@@ -300,7 +300,8 @@ export class Draw {
     }
     this.#fireEvent({
       type: 'annotationselect',
-      annotationId: group.id()
+      annotationId: group.id(),
```

Comment:
I forgot to say that events properties are all lower case in dwv, could you update them?